File: app/src/main/java/com/example/getfood/ui/orderdetail/OrderDetailMvpView.java
package com.example.getfood.ui.orderdetail;

import android.content.Context;

import com.example.getfood.ui.base.BaseView;
import com.fazemeright.canteen_app_models.models.FullOrder;

public interface OrderDetailMvpView extends BaseView {

    Context getContext();

    void bindOrderDetailAdapter(FullOrder updatedOrder);

    void onDatabaseError(Error error);

    void onRatingUpdatedSuccessfully();

    void onRatingUpdateFailed(Error error);
}

File: app/src/main/java/com/example/getfood/ui/orderdetail/OrderDetailMvpView.kt
package com.example.getfood.ui.orderdetail

import com.example.canteen_app_models.models.FullOrder
import com.example.getfood.ui.base.BaseView

interface OrderDetailMvpView : BaseView {

    fun bindOrderDetailAdapter(updatedOrder: FullOrder)
    fun onDatabaseError(error: Error?)
    fun onRatingUpdatedSuccessfully()
    fun onRatingUpdateFailed(error: Error)
}